neo/bin/prompt.py

```diff
@@ -40,19 +44,19 @@
     ShowUnspentCoins
 from neo.Prompt.Utils import get_arg
 from neo.Prompt.InputParser import InputParser
-from neo.Settings import settings, DIR_PROJECT_ROOT, PrivnetConnectionError
+from neo.Settings import settings, PrivnetConnectionError
 from neo.UserPreferences import preferences
 from neocore.KeyPair import KeyPair
 from neocore.UInt256 import UInt256
 
 # Logfile settings & setup
-LOGFILE_FN = os.path.join(DIR_PROJECT_ROOT, 'prompt.log')
+LOGFILE_FN = os.path.join(os.getcwd(), 'prompt.log')
 LOGFILE_MAX_BYTES = 5e7  # 50 MB
 LOGFILE_BACKUP_COUNT = 3  # 3 logfiles history
 settings.set_logfile(LOGFILE_FN, LOGFILE_MAX_BYTES, LOGFILE_BACKUP_COUNT)
 
 # Prompt history filename
-FILENAME_PROMPT_HISTORY = os.path.join(DIR_PROJECT_ROOT, '.prompt.py.history')
+FILENAME_PROMPT_HISTORY = os.path.join(os.getcwd(), '.prompt.py.history')
```

Comment:
I think the history should also be in `~/.neopython/.prompt.py.history`, else it would create a new history file wherever it is executed